src/App.css

```diff
@@ -63,10 +69,25 @@ hr {
   list-style: none;
 }
 
+.Title {
```

Comment:
- [ ] Kindly make sure your Styling is EXACTLY the same as the one provided on [Zeplin](https://zeplin.io/) just like the previous reviewer said, you should follow (the layout, colors, padding, margin...). check here the diference

Your design | The template 
-----|-----
<img width="1400" alt="image" src="https://github.com/Eccb7/book-store/assets/65068771/b882f0a2-fd5e-437b-bec1-4a832c50ef9a"> | <img width="1073" alt="image" src="https://github.com/Eccb7/book-store/assets/65068771/76904bb6-aeaf-4ece-8367-75464f2fb3d6">
<img width="1306" alt="image" src="https://github.com/Eccb7/book-store/assets/65068771/8bb68b62-b955-46c2-852c-5f184286d5e6"> | <img width="1053" alt="image" src="https://github.com/Eccb7/book-store/assets/65068771/1ea0b5b7-cbc9-4ffc-918a-0d8857b520c2">